src/components/flow/routers/dial/DialRouterForm.tsx

```diff
@@ -35,6 +38,30 @@ export default class DialRouterForm extends React.Component<RouterFormProps, Dia
     });
   }
 
+  private handleDialLimitUpdated(value: string): void {
+    const dialLimit = validate(
+      i18n.t('forms.dial_limit_seconds', 'Dialing time limit (seconds)'),
+      value,
+      [Numeric]
+    );
+    this.setState({
+      dialLimit: dialLimit,
+      valid: this.state.valid && !hasErrors(dialLimit)
+    });
+  }
+
+  private handleCallLimitUpdated(value: string): void {
+    const callLimit = validate(
+      i18n.t('forms.call_limit_seconds', 'Dialing time limit (seconds)'),
```

Comment:
`Call time limit (seconds)` ?

Reply:
oof, yes, that's a silly typo, thank you for the catch!